packages/devtools_app/test/evaluate_test.dart

```diff
@@ -0,0 +1,205 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
```

Comment:
nit: can we rename this file to `debugger_evalution_test.dart` so it is clear that this test is testing functionality in the debugger